ministryofjustice/analytics-platform-control-panel
# Add airflow to helm tools

## What

This PR adds the functionality to deploy a loc al version of the helm chart `airflow-sqlite`.

It is deployed in the users namespace and uses the `auth-proxy` for authentication.

## How to review

1. Go to tools
2. Click to deploy airflow
3. Click the link to see if airflow is accessible

https://trello.com/c/utqhVsoG
